community/neo4j/src/test/java/upgrade/TestMigrateToDenseNodeSupport.java

```diff
@@ -126,35 +124,6 @@ public void before() throws Exception
     }
 
     @Test
-    @Ignore( "Used for creating the dataset, using the previous store version" )
```

Comment:
Perhaps we can delete this test class completely? Doesn't seem relevant anymore.